generator/template/ready-to-use-section-types/sections/base/components/wysiwyg.vue

```diff
@@ -9,7 +9,7 @@ import "quill/dist/quill.snow.css";
 import "quill/dist/quill.bubble.css";
 
 import {quillEditor} from "vue-quill-editor";
-import {globalFileUpload} from "@geeks-solutions/vue-sections";
+import {globalFileUpload} from "@geeks.solutions/nuxt-sections/lib/src/utils";
```

Comment:
if you are hardcoding this here it means ONLY the nuxt-sections plugin can be installed and not the vue-sections one so you should update the readme accordingly

Reply:
this component only uses nuxt-sections, how will it work if vue-sections was installed ?

Reply:
It will work for both, for vue sections it can work by following the instruction in the README file under the section title `#### If Vue js is selected from the first question:`

* If you answered yes for adding 3 ready to use section types, you should find all declarations of `@geeks.solutions/nuxt-sections/lib/src/utils` (`wysiwyg.vue` under `sections/base/components/` and `nftSection.vue` under `sections/forms/`) and replace them by `@geeks.solutions/vue-sections` for the sections to work correctly